pac4j-core/src/main/java/org/pac4j/core/context/JEEContext.java

```diff
@@ -165,15 +165,7 @@ public String getFullRequestURL() {
 
     @Override
     public void addResponseCookie(final Cookie cookie) {
```

Comment:
I guess this could be moved as a default implementation of the `addResponseCookie` method in the `WebContext` interface.